### PR TITLE
Back to vue2

### PR DESCRIPTION
**Pull request by rleguen-broadsign**
Going back to Vue2
Installing vuetify
Installing @testing-library/vue
Writing some basic unit tests for vue and vuetify components, just to show we can

### CODE REVIEW

File: tests/unit/App.spec.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+import App from "@/App";
+import HelloWorld from "@/components/HelloWorld.vue";
+
+// from https://github.com/testing-library/vue-testing-library/blob/master/src/__tests__/vuetify.js
+
+// Custom container to integrate Vuetify with Vue Testing Library.
+// Vuetify requires you to wrap your app with a v-app component that provides
+// a <div data-app="true"> node.
+const renderWithVuetify = (component, options, callback) => {
```
Comment (rleguen-broadsign) [inline tests/unit/App.spec.js:15]: Will probably want to make a util method?

File: LICENSE [hunk 2]
```diff
@@ -1,21 +0,0 @@
-MIT License
```
Comment (rleguen-broadsign) [inline LICENSE]: Maybe we should keep this file

File: tests/unit/App.spec.js (hunk 1 above)
Comment (rleguen-broadsign) [reply tests/unit/App.spec.js:15]: Is this really necessary? https://iobio.io/user_guides/2019/05/20/vue-js-unit-testing-with-jest/

**Review by kseniya-baluk-broadsign — APPROVED** [review]
Looks good to me!

**Pull request merged**